web/docs/_static/openapi.yaml

```diff
@@ -206,7 +206,67 @@ paths:
           description: >-
             Invalid token OR expired signature
             OR no token given OR no password given.
-
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/message'
+  /user/change_email/request:
+    post:
+      summary: >-
+        Given the current identity's password and a new e-mail,
```

Comment:
Current identity is something that has not been referenced before, it requires some explanation.